### ISSUE TITLE
Crash in ViewPager

### ISSUE THREAD

**Issue by JEuler** [4eRTuk/audioview#8]
How I can setup AudioView in ViewPager? Should I use AudioView2? Or can go with AudioView? 
`android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.` I am receiving this when I am trying to go with AudioView inside FragmentStatePagerAdapter.

**Comment by JEuler** [comment]
Full stack-trace
```2019-04-17 13:44:11.594 9986-10642/com.mindful_inc.sparkler.qa E/AndroidRuntime: FATAL EXCEPTION: Thread-13
    Process: com.mindful_inc.sparkler.qa, PID: 9986
    android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:7695)
        at android.view.ViewRootImpl.requestLayout(ViewRootImpl.java:1212)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.RelativeLayout.requestLayout(RelativeLayout.java:360)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.RelativeLayout.requestLayout(RelativeLayout.java:360)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.RelativeLayout.requestLayout(RelativeLayout.java:360)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.ScrollView.requestLayout(ScrollView.java:1533)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.RelativeLayout.requestLayout(RelativeLayout.java:360)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.view.View.requestLayout(View.java:23089)
        at android.widget.TextView.checkForRelayout(TextView.java:8908)
        at android.widget.TextView.setText(TextView.java:5730)
        at android.widget.TextView.setText(TextView.java:5571)
        at android.widget.TextView.setText(TextView.java:5528)
        at com.keenfin.audioview.AudioView$2.run(AudioView.java:100)
        at java.lang.Thread.run(Thread.java:764)```

**Comment by JEuler** [comment]
So, it is trying to set 
`mTime.setText(getTrackTime());` and telling me the Thread is wrong. How I can handle this correctly?

**Comment by 4eRTuk** [comment]
I think I fixed this problem in f07dbef768d987dcffdc6c925b6bcbbb562d964f. Also added Travis CI support so it would be great to add some tests.

**Comment by JEuler** [comment]
Hm, will see what I can do with tests. And thank you for bugfix (will check it soon). Great library!
